extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -45,17 +77,20 @@ describes.realWin(
       config = defaultInlineConfig
     ) {
       pageElement.appendChild(
-        <amp-story-shopping-config layout="nodisplay" src={src}>
+        <amp-story-shopping-attachment layout="nodisplay" src={src}>
           <script type="application/json">{JSON.stringify(config)}</script>
-        </amp-story-shopping-config>
+        </amp-story-shopping-attachment>
+      );
+
+      return getShoppingConfig(
+        pageElement.getElementsByTagName('amp-story-shopping-attachment')[0]
       );
-      return getShoppingConfig(pageElement);
     }
 
     it('throws on no config', async () => {
       expectAsyncConsoleError(async () => {
         expect(() => {
-          pageElement.appendChild(<amp-story-shopping-config />);
+          pageElement.appendChild(<amp-story-shopping-attachment />);
```

Comment:
You should evaluate an element different from `pageElement`. Same thing here, put `<amp-story-shopping-attachment />` in an intermediate variable, and pass that to `getShoppingConfig`.

Reply:
Yes, sounds good.